Kudu.Core/Deployment/DeploymentManager.cs

```diff
@@ -836,7 +837,10 @@ private string GetDeploymentManifestPath(string id)
 
         private string GetLogPath(string id, bool ensureDirectory = true)
         {
-            return Path.Combine(GetRoot(id, ensureDirectory), LogFile);
+            var logPath = Path.Combine(GetRoot(id, ensureDirectory), XmlLogFile);
+            return FileSystemHelpers.FileExists(logPath)
+                ? logPath
+                : Path.Combine(GetRoot(id, ensureDirectory), TextLogFile);
```

Comment:
Mind adding comments on what scenario either Xml or Text logger will be used.  Looking at codes checking File exist is not obvious enough.